source/channels/message-priority.rst

```diff
@@ -0,0 +1,46 @@
+Set message priority (beta)
```

Comment:
SInce it's in beta, how do people give feedback (ie the give feedback button in the pop-over)